web/files/files.go

```diff
@@ -208,46 +218,61 @@ func ModificationHandler(c *gin.Context) {
 	}
 
 	patchData := container.Data
-	docType, err := vfs.ParseDocType(patchData.Type)
+
+	fileID := c.Param("file-id")
+
+	var docType vfs.DocType
+	if fileID == MetadataPath {
```

Comment:
The reason I did not do that is that it appears that Gin does not allow it... I get a panic `wildcard route ':file-id' conflicts with existing children in path '/files/:file-id`